include/wlr/types/wlr_linux_dmabuf.h

```diff
@@ -0,0 +1,42 @@
+#ifndef WLR_TYPES_WLR_SIMPLE_DMABUF_H
+#define WLR_TYPES_WLR_SIMPLE_DMABUF_H
+
+#define MAX_LINUX_BUFFER_PLANES 4
+
+struct wlr_dmabuf_buffer_attribs {
+	/* set via params_add */
+	int n_planes;
+	uint32_t offset[MAX_LINUX_BUFFER_PLANES];
+	uint32_t stride[MAX_LINUX_BUFFER_PLANES];
+	uint64_t modifier[MAX_LINUX_BUFFER_PLANES];
+	int fd[MAX_LINUX_BUFFER_PLANES];
```

Comment:
You could maybe instead use a struct for `offset`, `stride`, `modifier` and `fd`?

Reply:
It's much more consistent for other dmabuf APIs if it stays this way.